.drone.yml

```diff
@@ -104,6 +104,7 @@ steps:
       repo: us.gcr.io/kubernetes-dev/oncall
       dockerfile: engine/Dockerfile
       context: engine/
+      target: prod
```

Comment:
specifying `target` of `prod` simply means "don't install the sqlite/mysql/postgres clients into the image"